dbt_sugar/core/ui/cli_ui.py

```diff
@@ -224,11 +225,32 @@ def _iterate_through_columns(
                     ).ask()
                     if tests:
                         results[column]["tests"] = tests
+
+            # kick in the tags flow
+            if ask_for_tags:
+                wants_to_add_tags = questionary.confirm(
+                    message="Would you like to add any tags?"
+                ).ask()
+                if wants_to_add_tags:
+                    tags = questionary.text(message="Prodive a comma-separated list of tags").ask()
+                    tags = cls.__split_comma_separated_str(tags)
+                    if tags:
+                        results[column]["tags"] = tags
             # remove the column if no info has been given (no tests, and no description).
             if not results[column]:
                 _ = results.pop(column)
         return results
 
+    @staticmethod
+    def __split_comma_separated_str(tags: str) -> List[str]:
+        _tags = []
+        if isinstance(tags, str):
+            _tags = tags.split(",")
+            if isinstance(tags, list):
```

Comment:
it never will be a list, because you of this:
_tags = tags.split(",")

Reply:
Ah so I can skip that check even if someone only writes say "PII" it will be turned into a list of 1 item?

Reply:
Oh yeah you're right. Nice! I'm going to simplify this then